src/test/scala/com/campudus/tableaux/api/structure/CreateColumnTest.scala

```diff
@@ -223,6 +224,66 @@ class CreateColumnTest extends TableauxTestBase {
     }
   }
 
+  @Test
+  def createNumberColumnWithCustomDecimalDigits(implicit c: TestContext): Unit = {
```

Comment:
[MINOR] Für mich sieht es hier eher nach `createNumberColumnWithDefaultDecimalDigits` aus.

Reply:
Meine Welt ist soweit in Ordnung 👍🏻